nifi-nar-bundles/nifi-mqtt-bundle/nifi-mqtt-processors/src/test/java/org/apache/nifi/processors/mqtt/TestPublishMQTT.java

```diff
@@ -65,4 +67,16 @@ public void init() {
         topic = "testTopic";
         testRunner.setProperty(PublishMQTT.PROP_TOPIC, topic);
     }
+
+
+    @AfterEach
+    public void tearDown() {
+        final File folder =  new File("./target");
+        final File[] files = folder.listFiles((dir, name) -> name.matches( "moquette_store.mapdb.*" ));
+        for ( final File file : files ) {
+            if ( !file.delete() ) {
+                System.err.println( "Can't remove " + file.getAbsolutePath() );
+            }
+        }
+    }
```

Comment:
With recent refactoring of this module, this method and related changes in this file should not be necessary.